cdflow_commands/plugins/ecs.py

```diff
@@ -97,9 +98,15 @@ def _deploy_factory():
         )
         s3_bucket_factory = S3BucketFactory(boto_session, account_id)
         s3_bucket = s3_bucket_factory.get_bucket_name()
-        write_terragrunt_config(
-            metadata.aws_region, s3_bucket, environment_name, component_name
+
+        lock_table_factory = LockTableFactory(boto_session)
```

Comment:
This doesn't acquire the lock it just sets up the table which Terraform then uses.

We do duplicate it, which is annoying. There's other bits duplicated between them which we should look at consolidating.